CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 ## 3.2.0.0
-* Remove interactivity utils
+
+### Features
 * Add y-axis labels

```

Comment:
y-axis data bucket already existed before, so better to remove this point "Add y-axis labels"